features/steps/core.py

```diff
@@ -66,6 +67,22 @@ def set_config(context, config_file):
             cf.write("version: {}".format(__version__))
 
 
+@when('we open the editor and enter')
+@when('we open the editor and enter {text}')
+def open_editor_and_enter(context, text=""):
+    text = (text or context.text)
+    print("open_editor_and_enter called")
+    def _mock_editor_function(command):
+        print("_mock_editor_function called")
+        tmpfile = command.split()[-1]
```

Comment:
command is already a list
```suggestion
        tmpfile = command[-1]
```